liquidhaskell-boot/src/Language/Haskell/Liquid/Types/Names.hs

```diff
@@ -298,6 +300,11 @@ getLHGHCName :: LHName -> Maybe GHC.Name
 getLHGHCName (LHNResolved (LHRGHC n) _) = Just n
 getLHGHCName _ = Nothing
 
+qualifyLHName :: LHName -> Symbol
+qualifyLHName (LHNResolved (LHRGHC n) _) = qualifiedNameSymbol n
+qualifyLHName (LHNResolved _ s) = s
+qualifyLHName (LHNUnresolved _ s) = s
```

Comment:
This seems to do the same in intent as `lhNameToResolvedSymbol`.

Reply:
Not really, it only qualifies GHC symbols, whereas `lhNameToResolvedSymbol` only qualifies logic symbols.

Reply:
`lhNameToResolvedSymbol` qualifies LHNames. `symbol` does produce qualified names for the names that come in LHNames at least. This `qualifyLHName` only qualifies `LHRGHC` names.

Reply:
You're right, nevermind :)